tests/data/test_portfolio_profitability.py

```diff
@@ -122,50 +121,56 @@ async def test_get_current_crypto_currencies_values(backtesting_trader):
 
     exchange_manager.client_symbols.append("XLM/BTC")
     exchange_manager.client_symbols.append("XRP/BTC")
-    await portfolio_profitability.handle_mark_price_update("XRP/BTC", 0.005)
-    exchange_manager.client_symbols.append("NANO/BTC")
-    await portfolio_profitability.handle_mark_price_update("NANO/BTC", 0.05)
-    exchange_manager.client_symbols.append("BTC/USDT")
-
-    assert await portfolio_profitability.get_current_crypto_currencies_values() == {
-        'BTC': 1,
-        'ETH': 0,
-        'XRP': 0.005,
-        'NANO': 0.05,
-        'XLM': 0,
-        'USDT': 0
-    }
-    xlm_btc_price = random_price(max_value=0.05)
-    await portfolio_profitability.handle_mark_price_update("XLM/BTC", xlm_btc_price)
-    assert await portfolio_profitability.get_current_crypto_currencies_values() == {
-        'BTC': 1,
-        'ETH': 0,
-        'XRP': 0.005,
-        'NANO': 0.05,
-        'XLM': xlm_btc_price,
-        'USDT': 0
-    }
-    usdt_btc_price = random_price(max_value=0.01)
-    await portfolio_profitability.handle_mark_price_update("BTC/USDT", usdt_btc_price)
-    assert await portfolio_profitability.get_current_crypto_currencies_values() == {
-        'BTC': 1,
-        'ETH': 0,
-        'XRP': 0.005,
-        'NANO': 0.05,
-        'XLM': xlm_btc_price,
-        'USDT': 1 / usdt_btc_price
-    }
-    eth_btc_price = random_price(max_value=1)
-    exchange_manager.client_symbols.append("ETH/BTC")
-    await portfolio_profitability.handle_mark_price_update("ETH/BTC", eth_btc_price)
-    assert await portfolio_profitability.get_current_crypto_currencies_values() == {
-        'BTC': 1,
-        'ETH': eth_btc_price,
-        'XRP': 0.005,
-        'NANO': 0.05,
-        'XLM': xlm_btc_price,
-        'USDT': 1 / usdt_btc_price
-    }
+    if not os.getenv('CYTHON_IGNORE'):
```

Comment:
:disappointed_relieved:

Reply:
Due to `missing_currency_data_in_exchange` ?

Reply:
Due to it being private yes